MapboxCoreNavigation/RouteController.swift

```diff
@@ -198,6 +198,7 @@ open class RouteController: NSObject {
             events.initialize(withAccessToken: mapboxAccessToken, userAgentBase: "MapboxEventsNavigationiOS", hostSDKVersion: String(describing: Bundle(for: RouteController.self).object(forInfoDictionaryKey: "CFBundleShortVersionString")!))
             events.disableLocationMetrics()
             events.sendTurnstileEvent()
+            UIDevice.current.isBatteryMonitoringEnabled = true
```

Comment:
Should we also stop monitoring? Perhaps in init and deinit of `RouteController` or `NavigationLocationManager`?